src/vde_device.c

```diff
@@ -101,7 +89,7 @@ static void close_device(void) {
 }
 
 static bool read_packet(vpn_packet_t *packet) {
-	ssize_t lenin = (ssize_t) plug.vde_recv(conn, DATA(packet), MTU, 0);
+	ssize_t lenin = vde_recv(conn, DATA(packet), MTU, 0);
```

Comment:
I just see now that according to the manpage, anything less than 14 bytes should be discarded.

Reply:
I added some more message logging below. Please do let me know if ``event_exit()`` should be called in those cases. I feel they are recoverable errors, so I decided not to call it.

Reply:
I think those are recoverable as well. Anyway, there is already some code that will cause the event loop to exit if `read_packet()` returns `false` many times in succession.